### PR TITLE
fix: Don't print import certificate message in single-host mode

### PR DESCRIPTION
**Pull request by tolusha**
Signed-off-by: Anatolii Bazko <abazko@redhat.com>

### What does this PR do?
Don't print import certificate message in single-host mode

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/18082

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [x] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [x] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

**Review by AndrienkoAleksandr — APPROVED** [review]

File: src/tasks/installers/common-tasks.ts [hunk 1]
```diff
@@ -138,12 +138,16 @@ export function retrieveCheCaCertificateTask(flags: any): Listr.ListrTask {
...
       const cheCaCert = await che.retrieveCheCaCert(flags.chenamespace)
       if (cheCaCert) {
         const targetFile = await che.saveCheCaCert(cheCaCert)
 
         task.title = `${task.title}... done`
-        ctx.highlightedMessages.push(getMessageImportCaCertIntoBrowser(targetFile))
+        const cheConfigMap = await kube.getConfigMap('che', flags.chenamespace)
+        if (cheConfigMap && cheConfigMap.data && cheConfigMap.data.CHE_INFRA_KUBERNETES_SERVER__STRATEGY !== 'single-host') {
+          ctx.highlightedMessages.push(getMessageImportCaCertIntoBrowser(targetFile))
+        }
```
Comment (mmorhun) [inline src/tasks/installers/common-tasks.ts:150]: Does it make sense to create an util method to get server strategy?

File: src/api/kube.ts [hunk 2]
```diff
@@ -525,6 +525,18 @@ export class KubeHelper {
...
 
+  async getConfigMapValue(name: string, namespace: string, key: string): Promise<string | undefined> {
+    const k8sCoreApi = KubeHelper.KUBE_CONFIG.makeApiClient(CoreV1Api)
+    try {
+      const { body } = await k8sCoreApi.readNamespacedConfigMap(name, namespace)
+      if (body.data) {
+        return body.data[key]
+      }
+    } catch {
+      return
```
Comment (AndrienkoAleksandr) [inline src/api/kube.ts:536]: Maybe in the tslint.json we should apply: `"no-empty": [true, "allow-empty-catch", "allow-empty-functions"]` and change code:

```
} catch {}
```
To fix ` Opening brace cannot be followed by empty line`

Comment (AndrienkoAleksandr) [reply src/api/kube.ts:536]: But up to you.

**Pull request merged**